File: Jukebox/Jukebox/Jukebox/Models/Song.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity.Spatial;


namespace Jukebox
{


    [Serializable]
    public partial class Song
    {
        public int ID { get; set; }

        public string Name { get; set; }

        public string Author { get; set; }

        public string Genre { get; set; }

        public int DurationMinutes { get; set; }

        public int DurationSeconds { get; set; }

        public List<int> Playlists_ID { get; set; }

    }
}

File: Jukebox/Jukebox/Song.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

Partial Public Class Song
    Public Property ID As Integer

    Public Property Name As String

    Public Property Author As String

    Public Property DurationMinutes As Integer

    Public Property DurationSeconds As Integer

    Public Property Playlists_ID As Integer?

    Public Overridable Property Playlist As Playlist
End Class
